src/connection.cpp

```diff
@@ -76,11 +76,15 @@ std::string get_info_or_empty(connection_ptr const& p, short type) {
 // [[Rcpp::export]]
 Rcpp::List connection_info(connection_ptr const& p) {
   SQLUINTEGER getdata_ext;
+  SQLUINTEGER owner_usage;
   try {
     getdata_ext =
         (*p)->connection()->get_info<SQLUINTEGER>(SQL_GETDATA_EXTENSIONS);
+    owner_usage =
+        (*p)->connection()->get_info<SQLUINTEGER>(SQL_OWNER_USAGE);
```

Comment:
I see on the linked Microsoft docs that `SQL_OWNER_USAGE` was deprecated in favor of `SQL_SCHEMA_USAGE` in ODBC 3.x, released in 1995. As someone relatively new to using ODBC, a question: do we generally aim to support all ODBC versions or are we better off to use the newer keyword in anticipation of the older one being fully deprecated? From what I can tell, the deprecation cycle for ODBC is loooong, so fine by me to use the older keyword as we are here if we anticipate it will be supported for a long time.

Reply:
Hey Simon - I went back on forth on this one, but your comment nudged me in the direction of `SCHEMA` in favor of `OWNER`.

Thanks!